model/Core/Properties/requirementRationale.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# requirementRationale
+
+## Summary
+
+Text used to define the rationale or additional information.
+
+## Description
+
+Additional detail used to define the reason/rationale, why this requirement is there or give some additional information, needed to understand the requirement. Usually less formal than the requirement wording of the RequirementStatement.
```

Comment:
```suggestion
Additional detail used to define the reason or justification for the existence of the requirement. The rationale is usually less formal than the wording of the requirement statement itself.
```

Reply:
This my last review comment became Outdated due to the other changes in this PR. I think it is a good improvement, so I restored it here.